lib/virt_autotest/common.pm

```diff
@@ -165,6 +165,26 @@ if (get_var("REGRESSION", '') =~ /xen/) {
             location => 'http://mirror.suse.cz/install/SLP/SLE-12-SP5-Server-LATEST/x86_64/DVD1/',
             linuxrc => 'ifcfg="eth0=192.168.122.114/24,192.168.122.1,192.168.122.1"',
         },
+        sles15sp4PV => {
+            name => 'sles15sp4PV',
+            autoyast => 'autoyast_xen/sles11sp4PVx32_PRG.xml',
+            extra_params => '--os-variant sle15-unknown',    # problems after kernel upgrade
```

Comment:
sle15sp4 should be suppored in infodb as the bug was fixed on 15sp3/2/1, https://bugzilla.suse.com/show_bug.cgi?id=1197958

Reply:
Yes, I noticed but we have to cover sle12.3+ hosts. It's not defined in 12.3 infodb, isn't it?